plugin.video.vrt.nu/resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -99,15 +105,10 @@ def get_global_setting(self, setting):
     def get_proxies(self):
         usehttpproxy = self.get_global_setting('network.usehttpproxy')
```

Comment:
Is usehttpProxy a boolean? otherwise i think its better to do if not usehttpproxy

Reply:
It is a boolean and I am indifferent to how to do this, although this makes it clear that it is a boolean IMO.